code/nodes/predictParams/predictParams.q

```diff
@@ -1,27 +1,34 @@
-\d .automl
-
+// code/nodes/predictParams/predictParams.q - Predict params
+// Copyright (c) 2021 Kx Systems Inc
+//
 // Collect all the parameters relevant for the generation of reports/graphs etc
 //   in the prediction step such they can be consolidated into a single node 
 //   later in the workflow
 
+\d .automl
+
 // @kind function
 // @category node
-// @fileoverview Collect all relevant parameters from previous prediction steps
+// @desc Collect all relevant parameters from previous prediction steps
 //   to be consolidated for report/graph generation
 // @param bestModel {<} The best model fitted 
-// @param hyperParmams {dict} Hyperparameters used for model (if any)
-// @param modelName {str} Name of best model
+// @param hyperParmams {dictionary} Hyperparameters used for model (if any)
+// @param modelName {string} Name of best model
 // @param testScore {float} Score of model on testing data
-// @param modelMetaData {dict} Meta data from finding best model
-// @return {dict} Consolidated parameters to be used to generate reports/graphs 
-predictParams.node.function:{[bestModel;hyperParams;modelName;testScore;analyzeModel;modelMetaData]
+// @param modelMetaData {dictionary} Meta data from finding best model
+// @return {dictionary} Consolidated parameters to be used to generate 
+//   reports/graphs 
+predictParams.node.function:{[bestModel;hyperParams;modelName;testScore;
```

Comment:
Can be a single line if it's a function input (even if its over 80 chars)

Reply:
done as above